src/Sulu/Bundle/AdminBundle/DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,21 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('icon_sets')
+                    ->useAttributeAsKey('name')
+                    ->normalizeKeys(false)
+                    ->prototype('array')
+                        ->children()
+                            ->scalarNode('provider')->end()
+                            ->arrayNode('options')
+                                ->children()
```

Comment:
not sure if we can specify which options are needed for which provider. maybe @alexander-schranz  knows more about this 🙂